kinto/core/__init__.py

```diff
@@ -154,6 +171,9 @@ def add_api_capability(config, identifier, description='', url='', **kw):
     # Add CORS settings to the base kinto.core Service class.
     Service.init_from_settings(settings)
 
+    # Use the project name as the main logger name (Logger field in MozLog).
+    JsonLogFormatter.init_from_settings(settings)
```

Comment:
If it is the default, why do we need to add it in the config?

Reply:
I didn't understand this question